manifests/kustomize/README.md

```diff
@@ -118,6 +78,23 @@ kubectl -n <KFP_NAMESPACE> delete deployment metadata-deployment
 kubectl -n <KFP_NAMESPACE> delete service metadata-service
 ```
 
+## Uninstall
+
+```
+### 1. namespace scoped
+# Depends on how you installed it:
+kubectl kustomize env/dev | kubectl delete -f -
```

Comment:
Although existing documentation here is written like this, I'd prefer the shorthand versions
```
kubectl apply -k env/gcp-dev
kubectl delete -k env/gcp-dev
```

they are equivalent (kubeflow.org website uses this pattern)

Reply:
I had another PR on the way, let's solve that in that PR.

Reply:
sure, SGTM